### PR TITLE
gridly: Make segment-range an openlilylib option

### PR DESCRIPTION
**Pull request by uliska**
I think this makes stuff somewhat clearer.

Closes https://github.com/openlilylib/openlilylib/issues/65 (see for description)

### CODE REVIEW

**Comment by uliska** [comment]
Ah, I changed from using `\setOption` to a dedicated `\gridSetRange` command because that can provide the type check.

**Comment by Cecca** [comment]
Thank you Urs, this makes things easier, indeed. I think that, before merging this into GridLY, I should open a `gridly/development` branch to continue development, so to keep a stable version on `master`.

File: ly/gridly/__main__.ily [hunk 1]
```diff
@@ -44,6 +44,9 @@
    (closing #:init-keyword #:closing
             #:getter cell:closing))
 
+%%% Configuration
+\registerOption gridly.segment-range #'all
```
Comment (Cecca) [inline ly/gridly/__main__.ily:48]: Maybe we can put this in `__init__.ily`?

Comment (uliska) [inline ly/gridly/__main__.ily:48]: Yes, that's where it belongs.  
I've not fully internalized the structure of the lib yet.

**Comment by uliska** [comment]
OK, then I suggest you create that branch (as I don't know if you have 
added anything to it in the meantime), and I'll redo the pull request, 
incorporating your **init**.ily suggestion

**Comment by Cecca** [comment]
Ok, I created branch [gridly/development](https://github.com/openlilylib/openlilylib/tree/gridly/development). The only changes are a bump of the version number and the changes to the examples you made in PR #62.

Please, feel free to create another pull request.

**Pull request closed without merging**